bno055/connectors/I2CConnector.py

```diff
@@ -0,0 +1,11 @@
+from bno055.connectors.Connector import Connector
+from rclpy.node import Node
+
+
+class I2CConnector(Connector):
```

Comment:
renaming to just `I2C` might make more sense, no?
```suggestion
class I2C(Connector):
```

Reply:
done